src/hooks/useCheckbox.ts

```diff
@@ -48,7 +48,14 @@ export const getCheckboxCustomFields = (
     if (currentObject.items.enum) {
       items = currentObject.items.enum
     } else if (currentObject.items.type === 'string') {
-      items = currentObject.enum ? currentObject.enum : []
+      items = currentObject.enum
+        ? currentObject.enum.map(obj => {
```

Comment:
Move the function of this map to a module and give it a nice name so people understand what this code does and to avoid repetition